packages/core/package.json

```diff
@@ -41,15 +41,15 @@
     "typescript": "^4.5.5"
   },
   "dependencies": {
-    "@web3-onboard/common": "^2.0.7",
+    "@web3-onboard/common": "^2.1.0-alpha.2",
```

Comment:
Should the alpha be removed here for release?

Reply:
Good eye - Fixed!